README.md

```diff
@@ -31,7 +31,20 @@ This project uses [lerna](https://github.com/lerna/lerna) with [yarn workspaces]
 - `web-www`: Regen website based on [Gatsby](https://www.gatsbyjs.org/)
 - `web-auth`: React application used for Auth0 Custom Universal Login
 
-Install dependencies using:
+### Prerequisites
+
+#### Mac
+
+To set up system dependencies run the following: 
```

Comment:
maybe good to add that this is only needed if you don't already have python and/or vips installed